containers.go

```diff
@@ -312,22 +312,30 @@ func runContainer(daemon *docker.Client, id string, logger log15.Logger, logfile
 	// If we're the outer shell, log straight to stderr, nothing fancy
 	stdout := io.Writer(os.Stdout)
 	stream := io.Writer(os.Stderr)
+	var log, hookedR, hookedW *os.File
+	var fdsToClose []io.Closer
+	var err error
```

Comment:
The hookedW/hookedR/err declarations are leftovers from a previous version of this fix -- they're indeed not necessary, so I reverted them.